oremda/pipeline.py

```diff
@@ -136,6 +136,13 @@ def __init__(
     def id(self):
         return self._id
 
+    @property
+    def image_names(self):
+        return list(set(node.operator.image_name for node in self.nodes.values()))
```

Comment:
Just a nitpick, but since `set` is iterable, we can probably skip the conversion to `list`

Reply:
Sure. Really, the only reason I used `set()` here was to get rid of duplicates. I tend to use lists most of the time for iterables. But I can get rid of that here.